config/unicorn.conf.rb

```diff
@@ -1,15 +1,21 @@
 # frozen_string_literal: true
 
 # See http://unicorn.bogomips.org/Unicorn/Configurator.html
+discourse_path = File.expand_path(File.expand_path(File.dirname(__FILE__)) + "/../")
+
+enable_logstash_logger = ENV["ENABLE_LOGSTASH_LOGGER"] == "1"
+unicorn_stderr_path = "#{discourse_path}/log/unicorn.stderr.log"
+unicorn_stdout_path = "#{discourse_path}/log/unicorn.stdout.log"
 
-if (ENV["LOGSTASH_UNICORN_URI"] || "").length > 0
+if enable_logstash_logger
   require_relative "../lib/discourse_logstash_logger"
   require_relative "../lib/unicorn_logstash_patch"
-  logger DiscourseLogstashLogger.logger(uri: ENV["LOGSTASH_UNICORN_URI"], type: :unicorn)
+  FileUtils.touch(unicorn_stderr_path) if !File.exist?(unicorn_stderr_path)
```

Comment:
Create the file if it does not exists. This can happen if the logs directory is mounted with a host directory when launching the Docker container.